docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[release-notes-header-8.6.0]]
+== 8.6
+
+[discrete]
+[[release-notes-8.6.0]]
+=== 8.6.0
+
+[discrete]
+[[known-issue-8.6.0]]
+==== Known issues
+There are no known issues in 8.6.0.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+// end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.6.0]]
+==== Deprecations
+There are no deprecations in 8.6.0.
+
+
+[discrete]
+[[features-8.6.0]]
+==== Features
+* Allows you to add indicators to new or existing cases ({pull}145121[#145121]).
+* Adds the `is one of` operator to the *Add field* menu in Timeline ({pull}144988[#144988]).
+* Adds an "Add to timeline investigation" button to the User Risk Scores and Host Risk Scores cards on the Entity Analytics dashboard ({pull}144819[#144819]).
+* Provides the option to duplicate rules and their exceptions or rules only ({pull}144782[#144782]).
+* Improves the Rule Exceptions interface and allows you to export read-only exception lists ({pull}144383[#144383]).
+* Enables a technical preview of functionality that allows users to create runtime queries that can be parameterized from alert data or hard coded literal values. ({pull}145240[#145240]).
```

Comment:
We might want to check with @paulewing about how to communicate this feature, since it's pretty heavily in-development at this time. If we're in a hurry to merge the rest of the RNs soon, maybe omit this line while we get confirmation from Paul, then we can add it in later with a small separate PR?

Reply:
This sounds like a pretty clear and straight forward description of what the feature actually is, I think @paulewing is considering a few different singular names for the feature, i.e. "insight" or "investigation" or some others, but this literal description is 100% accurate and unlikely to change. More features will be added but the core functionality will remain the same.